# Let $S_{n}$ be the number of successes in $n$ Bernoulli trials with $p=0.8$

# Let $A_{n} = \frac{S_{n}}{n}$ be the average number of successes in the $n$-trials (i.e. the observed probabilty of success)

# In each case, give the value of the limit and the reason why:

## a) $\lim_{n\rightarrow \infty}P(A_{n}=0.8)$

## b) $\lim_{n\rightarrow \infty}P(0.7n < S_{n} < 0.9n)$

## c) $\lim_{n\rightarrow \infty}P(S_{n} < 0.8n + 0.8\sqrt{n})$

## d) $\lim_{n\rightarrow \infty}P(0.79 < A_{n} <0.81)$

____

# a)

# As $n\rightarrow \infty$, $A_{n}$ becomes a continuous random variable

# This means we measure the probability that $a \leq A_{n} \leq b$ since the probability that it equals any single value is infinitesimal 

# Therefore $\lim_{n\rightarrow \infty}P(A_{n}=0.8) = 0$

___

# b)

# $\lim_{n\rightarrow \infty}P(0.7n < S_{n} < 0.9n) = \lim_{n\rightarrow \infty}P(0.7 < A_{n} < 0.9)$

# The Law of Large Numbers states that $\lim_{n\rightarrow \infty} P\left ( \left |A_{n} - \mu \right | < \epsilon \right ) = 1$

# Therefore $\lim_{n\rightarrow \infty}P(0.7 < A_{n} < 0.9) = 1$

_____

# c)

# $\lim_{n\rightarrow \infty}P(S_{n} < 0.8n + 0.8\sqrt{n}) = \lim_{n\rightarrow \infty} P(A_{n} - 0.8 < \frac{0.8}{\sqrt{n}})$

# We know that $V(S_{n}) = npq \implies V(A_{n}) = \frac{pq}{n} = \frac{0.16}{n} \implies \sigma_{A_{n}} = \frac{0.4}{\sqrt{n}}$

# This means $\frac{0.8}{\sqrt{n}} = 2\sigma_{A_{n}}$

# The Central Limit Theorem says:

# $\lim_{n\rightarrow \infty} P(a\sigma \leq A_{n} - \mu \leq b\sigma) = \int_{a}^{b}\phi(x)dx$

# $\implies \lim_{n\rightarrow \infty} P(A_{n} - 0.8 < \frac{0.8}{\sqrt{n}}) = \int_{-\infty}^{2}\phi(x)dx$

In [1]:
from scipy.stats import norm

In [2]:
norm.cdf(2)

0.97724986805182079

# Therefore $\lim_{n\rightarrow \infty} P(A_{n} - 0.8 < \frac{0.8}{\sqrt{n}}) = 0.977$

____

# d)

# 1, for the same reason as b)